rasgotransforms/rasgotransforms/transforms/drop_columns/drop_columns.yaml

```diff
@@ -16,6 +16,10 @@ arguments:
     type: column_list
     description: A list of the columns from the dataset you want to drop. Any columns not in the exclude_cols list will be kept.
     is_optional: true
+  case_sensitive_exclude:
```

Comment:
could we name this something generic that can be used across templates. I can instantly see this expanding to all templates that invoke `get_columns`

Options?
case_sensitive_match
case_sensitive
case_sensitive_column_names